examples/ember-cli/todomvc/app/services/repo.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Service.extend({
+	lastId: 0,
+	findAll() {
+		if (this.data) {
+			return this.data;
```

Comment:
the people reading this won't have your nuance, it might encourage poorer practices. our current message is "always be get/settings"